tests/test_benchmark_free_energy.py

```diff
@@ -77,8 +77,7 @@ def test_benchmark_hif2a_single_topology(hif2a_single_topology_leg, enable_hrex)
             run_sims_hrex,
             initial_states,
             md_params,
-            n_frames_per_iter=5,
-            temperature=temperature,
+            n_frames_per_iter=1,
```

Comment:
why is temperature removed?

Reply:
Temperature is defined at the level of initial states for HREX. (Although we currently restrict the temperature to be the same for all states: https://github.com/proteneer/timemachine/blob/5951327be3b4e71a3325d0f1188d8c26ea1757a7/timemachine/fe/free_energy.py#L817-L821)